### DISCUSSION TITLE
able to view Bounding box or Motion boxes in the rtsp streams?

### DISCUSSION THREAD

**Discussion by RJ1002** [blakeblackshear/frigate#5980] [Ask A Question] [answered]
is there a way to make rtsp streams to have the Bounding box or Motion boxes in the stream? like what you can see in the debug view, but also be able to see those boxes in the rtsp stream? is there maybe a go2rtc settings?

**Comment by blakeblackshear** [comment, marked as answer]
No. It would require re-encoding a new stream and would take a lot of resources.
